inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx

```diff
@@ -184,8 +219,8 @@ export class Timeline extends React.Component<
     };
 
     /**
-    * Handles the change of number of frames available in the timeline.
-    */
+     * Handles the change of number of frames available in the timeline.
+     */

```

Comment:
Missing param

Reply:
Good catch. I'll update this and look for other places this is happening